src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -579,9 +584,22 @@ public override void ExecuteCmdlet()
                         }
                     }
                 });
+
+                WriteInformation($"[Announcements]{System.Environment.NewLine}Share your feedback regarding your experience with `Connect-AzAccount` at: https://aka.ms/azloginfeedback{System.Environment.NewLine}");
+                WriteInformation($"If you encounter any problem, please open an issue at: https://aka.ms/azpsissue{System.Environment.NewLine}");
             }
         }
 
+        private string GetLoginExperienceStatus()
```

Comment:
return boolean to avoid multiple magic strings (On and Off)
```suggestion
        private bool IsLoginExperienceV2Enabled()
```